File: Java/Iniciante/1021.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
  
  public static void main(String[] args) throws IOException {
    Scanner scanner = new Scanner(System.in);
  
    double valor = scanner.nextDouble();

    System.out.println("NOTAS:");
    System.out.println(String.format("%d nota(s) de R$ 100.00", (int)valor / 100));
    valor %= 100.00;

    System.out.println(String.format("%d nota(s) de R$ 50.00", (int)valor / 50));
    valor %= 50.00;

    System.out.println(String.format("%d nota(s) de R$ 20.00", (int)valor / 20));
    valor %= 20.00;

    System.out.println(String.format("%d nota(s) de R$ 10.00", (int)valor / 10));
    valor %= 10.00;

    System.out.println(String.format("%d nota(s) de R$ 5.00", (int)valor / 5));
    valor %= 5.00;

    System.out.println(String.format("%d nota(s) de R$ 2.00", (int)valor / 2));
    valor %= 2.00;

    valor *= 100.0;
    System.out.println("MOEDAS:");

    System.out.println(String.format("%d moeda(s) de R$ 1.00", (int)valor / 100));
    valor %= 100.00;

    System.out.println(String.format("%d moeda(s) de R$ 0.50", (int)valor / 50));
    valor %= 50.00;

    System.out.println(String.format("%d moeda(s) de R$ 0.25", (int)valor / 25));
    valor %= 25.00;

    System.out.println(String.format("%d moeda(s) de R$ 0.10", (int)valor / 10));
    valor %= 10.00;

    System.out.println(String.format("%d moeda(s) de R$ 0.05", (int)valor / 5));
    valor %= 5.00;

    System.out.println(String.format("%d moeda(s) de R$ 0.01", (int)valor / 1));
    
  }
}

File: C#/Iniciante/1021.cs
using System; 

class URI {

    static void Main(string[] args) { 

        double valor = Convert.ToDouble(Console.ReadLine());

        Console.WriteLine("NOTAS:");
        Console.WriteLine($"{Convert.ToInt32(Math.Floor(valor / 100))} nota(s) de R$ 100.00");
        valor %= 100;
        
        Console.WriteLine($"{Convert.ToInt32(Math.Floor(valor / 50))} nota(s) de R$ 50.00");
        valor %= 50;
        
        Console.WriteLine($"{Convert.ToInt32(Math.Floor(valor / 20))} nota(s) de R$ 20.00");
        valor %= 20;
        
        Console.WriteLine($"{Convert.ToInt32(Math.Floor(valor / 10))} nota(s) de R$ 10.00");
        valor %= 10;
        
        Console.WriteLine($"{Convert.ToInt32(Math.Floor(valor / 5))} nota(s) de R$ 5.00");
        valor %= 5;
        
        Console.WriteLine($"{Convert.ToInt32(Math.Floor(valor / 2))} nota(s) de R$ 2.00");
        valor %= 2;
        
        Console.WriteLine("MOEDAS:");
        Console.WriteLine($"{Convert.ToInt32(Math.Floor(valor / 1))} moeda(s) de R$ 1.00");
        valor %= 1;
        
        Console.WriteLine($"{Convert.ToInt32(Math.Floor(valor / 0.5))} moeda(s) de R$ 0.50");
        valor %= 0.5;
        
        Console.WriteLine($"{Convert.ToInt32(Math.Floor(valor / 0.25))} moeda(s) de R$ 0.25");
        valor %= 0.25;
        
        Console.WriteLine($"{Convert.ToInt32(Math.Floor(valor / 0.1))} moeda(s) de R$ 0.10");
        valor %= 0.1;
        
        Console.WriteLine($"{Convert.ToInt32(Math.Floor(valor / 0.05))} moeda(s) de R$ 0.05");
        valor %= 0.05;
        
        Console.WriteLine($"{Convert.ToInt32(valor / 0.01)} moeda(s) de R$ 0.01");

    }

}
